src/VirtoCommerce.ElasticSearch8.Data/Services/ElasticSearchResponseBuilder.cs

```diff
@@ -198,6 +199,22 @@ private static void AddRangeAggregationValues(AggregateDictionary searchResponse
                 var responseValueId = $"{aggregationResponse.Id}-{queryValueId}";
                 AddAggregationValues(aggregationResponse, responseValueId, queryValueId, searchResponseAggregations);
             }
+
+            TryAddAggregatioStatistics(searchResponseAggregations, aggregationResponse);
+        }
+
+        private static void TryAddAggregatioStatistics(AggregateDictionary searchResponseAggregations, AggregationResponse aggregationResponse)
```

Comment:
Typo in TryAddAggregatioStatistics.
Please install Reshaper or at least spell checker.